src/lib/rest/clientSocketHttp.cpp

```diff
@@ -333,13 +332,22 @@ std::string sendHttpSocket
   curl_easy_setopt(curl, CURLOPT_POSTFIELDS, (u_int8_t*) payload);
 
   // Set up URL
-  std::string url;
+  std::string url= "";
```

Comment:
Remember code syle, you shouldn't write:

```
if (...)
   foo;
else
   bar;
```

but

```
if (...) 
{
   foo;
}
else
{
   bar;
}
```
Only in new code (old code will be changed in the future as an independent style-fixing issue).